app/components/special-coverage/special-coverage-edit/special-coverage-edit-directive.js

```diff
@@ -66,8 +66,47 @@ angular.module('specialCoverage.edit.directive', [
           return promise;
         };
 
+        // Maps ID to Object for formatting (since only tunic_campaign_id stored on SpecialCoverage object)
+        $scope.tunicCampaignIdMapping = {};
+        $scope.campaignsRetrieved = function (response) {
+          response.data.results.forEach(function (result) {
+            $scope.tunicCampaignIdMapping[result.id] = result;
+          });
+          var tunicCampaignIds = [];
+          response.data.results.forEach(function (result) {
+            tunicCampaignIds.push(result.id);
+          });
+          return tunicCampaignIds;
+        };
+
+        $scope.getCampaign = function (tunicCampaignId) {
+          return $http.get('http://tunic.local/api/v1/campaign/' + tunicCampaignId + '/', {
```

Comment:
We'll have to come up with a strategy of how we want to get this data from py settings.  Maybe whatever we do should mirror how @kand is shipping settings down with the `bulbs-cms2` stuff to make that an easier transition